src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,12 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball [] balls = new Ball[10];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getBall();
+            System.out.println(balls[i].toString());
```

Comment:
```suggestion
            System.out.println(balls[i]);
```

We dont need to call toString() method, It will be called "under the hood" of System.out.println method